stable/mariadb/README.md

```diff
@@ -62,6 +63,8 @@ The following table lists the configurable parameters of the MariaDB chart and t
 | `replication.enabled`                     | MariaDB replication enabled                         | `true`                                                             |
 | `replication.user`                        | MariaDB replication user                            | `replicator`                                                       |
 | `replication.password`                    | MariaDB replication user password                   | _random 10 character alphanumeric string_                         |
+| `master.annotations[].key`                | key for the the annotation list item                |                                                    |
```

Comment:
Could you use `master.podAnnotations` as an object? So the use can specify additional annotations. Check how it's developed on the [Redis chart](https://github.com/helm/charts/tree/master/stable/redis)

Reply:
Thanks For the review juan131. `master.podAnnotations` in redis chart is surely simple way, but as I understand it takes multiple annotations as text string. While the way I am using it, it allows an array of annotations. To me array is more convenient what to specify, for example:

```
master:
  annotations:
  - key: endpoint0
    value: network0
  - key: endpoint1
    value: network1
```

Although, I am not too picky about this, and if redis way looks better to you we can do that way as well.

Thanks,
-Arvind

Reply:
Could you share an example of use? How will you indicate your annotations when running the command below?

```
helm install --name my-release stable/mariadb --set XXX=xxx, YYY=yyy, ZZZ=zzz 
```

Reply:
Hi @juan131 , sorry for late response. I got dragged in to other work. 

when using multiple array like I mentioned above, its better and easier to create cusom-values.yaml.  But still above mentioned annotations can be passed using --set as follows:

```
helm install --set master.annotations[0].key=endpoint0,master.annotations[0].value=network0,master.annotations[1].key=endpoint1,master.annotations[1].value=network1  stable/mariadb
```
output will be:
```
  template:
    metadata:
      annotations:
        endpoint0: 'network0'
        endpoint1: 'network1'
```
Does that solves your concern?

Thanks,

Reply:
Yes thanks!!! Please add the default value on the third column of the table.